test/serverless-artillery/serverless-artillery-test-users-api.yml

```diff
@@ -0,0 +1,17 @@
+# You can find great documentation of the possibilities at:
+# https://artillery.io/docs/
+config:
+  target: "https://nsc-fun-dev-usw2-thursday.azurewebsites.net/api"
```

Comment:
I might have to try this instead of linking directly to the specific function for what I need.